src/app/app.component.ts

```diff
@@ -117,41 +119,47 @@ export class AppComponent implements OnInit, OnDestroy {
   }
 
   getEpisodes(podcast: PodcastModel) {
+    const distObsv = [];
     podcast.doc.followItems('prx:stories', {
       per: podcast.doc.count('prx:stories'),
       filters: 'v4',
       sorts: 'released_at: desc, published_at: desc'
```

Comment:
Occurs to me that maybe this should just sort by published_at?  Though since the `/authorized/stories` endpoint doesn't filter to only-published, you'd still have to check for nulls.  And we've got some more CMS changes coming anyways.  Nevermind!

Reply:
honestly I probably dont need this sort since it's sorting in the episode reducer. yes, change is gonna come!